### PR TITLE
Step4 & Step5

### PR DESCRIPTION
**Pull request by F-0728**
## What
Created table.

## CHECKS :warning:

Please make sure you are aware of the following.

- [ ] **The changes in this PR doesn't have private information

### CODE REVIEW

File: db/items.db [hunk 1]
```diff
@@ -0,0 +1,11 @@
+CREATE TABLE category (
```
Comment (mshibuya) [inline db/items.db]: `categories` is correct, according to the instruction.

File: db/items.db [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+id INT PRIMARY KEY,
+name TEXT
+);
+CREATE TABLE items (
+id INT PRIMARY KEY,
+name TEXT,
+category_id INT,
+image_name TEXT,
+FOREIGN KEY (category_id) REFERENCES category(id)
```
Comment (mshibuya) [inline db/items.db]: It's a good practice to use the foreign key constraint 👍

File: go/app/main.go [hunk 3]
```diff
@@ -76,68 +80,91 @@ func addItem(c echo.Context) error {
...
+	db, err := sql.Open("sqlite3", dbPath)
 	if err != nil {
 		return err
 	}
+	defer db.Close()
 
-	var items Items
-	json.Unmarshal(jsonFile, &items)
+	// Query the database
+	rows, err := db.Query("SELECT name, category_id, image_name FROM items")
```
Comment (mshibuya) [inline go/app/main.go]: What we want here is only a single item. So retrieving all the items isn't efficient, especially if there's many items in the table.
There should be a way to retrieve only 1 item, so please check it out!

**Review by mshibuya — COMMENTED** [review]
Great progress! I've made some minor suggestions 👍

File: go/app/main.go (hunk 3 above)
Comment (F-0728) [reply go/app/main.go]: Thank you for the review!
Indeed we dont need to query the full table though we want only 1 item.
Now i modified the code, please check it.

**Pull request merged**